components/formats-api/src/loci/formats/FormatTools.java

```diff
@@ -199,6 +199,8 @@
 
   // -- Constants - versioning --
 
+  public static final String CREATOR = "OME Bio-Formats";
+
```

Comment:
My earlier comment is probably better here.  Could this include the version number?